profile/README.md

```diff
@@ -0,0 +1,28 @@
+# Building protocols for the future of the Internet
+
+Fission is building the _data_, _identity_ & _compute_ primitives that enable true local-first edge applications.
+
+## Working Groups
+
+We don't get to the future alone. Fission collaborates with other developers and organizations through a variety of working groups.
+
+- [UCAN (User Controlled Authorization Networks) Working Group](https://github.com/ucan-wg) - a trustless, secure, local-first, user-originated authorization and revocation scheme.
+- [WNFS (Webnative Filesystem) Working Group](https://github.com/wnfs-wg/) - a versioned, logged, programmable, flexibly secure distributed file system.
+- [IPVM (Interplanetary Virtual Machine) Working Group](https://github.com/ipvm-wg) - an open, decentralized, and local-first execution layer for IPFS.
+
+## Webnative
+
+Fission's true local-first, edge computing stack. Webnative empowers you to build fully distributed web applications with auth and storage without needing a complex backend.
+
+- [SDK](https://github.com/fission-codes/webnative) - the base SDK for building Webnative apps
+- [Guide](https://guide.fission.codes/) - documentation on using Webnative
+- [App Template](https://github.com/webnative-examples/webnative-app-template) - clone-and-go template for quickly building a web app using Webnative.
+- [Examples](https://github.com/webnative-examples) - example apps built with Webnative.
+- [Whitepaper](https://github.com/fission-codes/whitepaper) - original whitepaper that outlines the Webnative vision.
+
+## Connect with us!
+
+- [Follow us on Twitter](https://twitter.com/FISSIONcodes)
```

Comment:
I know we tweet links to the blog, but ... the blog has a lot of our best content these days - maybe deserves a link?

Reply:
Oh, great point!